modules/running-insights-operator-gather-web-console.adoc

```diff
@@ -6,20 +6,20 @@
 :_mod-docs-content-type: PROCEDURE
 
 [id="running-insights-operator-gather-web-console_{context}"]
-= Running an Insights Operator gather operation using the web console
-You can run an Insights Operator gather operation using the {product-title} web console.
+= Running an Insights Operator gather operation from the web console
+To collect data, you can run an Insights Operator gather operation by using the {product-title} web console.
 
 .Prerequisites
 
 * You are logged in to the {product-title} web console as a user with the `cluster-admin` role.
 
 .Procedure
 
-. Navigate to *Administration* -> *CustomResourceDefinitions*.
-. On the *CustomResourceDefinitions* page, use the *Search by name* field to find the *DataGather* resource definition and click it.
+. On the console, select *Administration* -> *CustomResourceDefinitions*.
+. On the *CustomResourceDefinitions* page, in the *Search by name* field, find the *DataGather* resource definition, and then click it.
```

Comment:
This step contains a few steps so some folk might suggest to break it out into substeps.